lib/Controller/OStatusController.php

```diff
@@ -122,14 +125,14 @@ public function subscribe(string $uri): Response {
 			}
 
 			$this->initialStateService->provideInitialState('social', 'serverData', [
-				'account'     => $actor->getAccount(),
+				'account' => $actor->getAccount(),
 				'currentUser' => [
-					'uid'         => $user->getUID(),
+					'uid' => $user->getUID(),
 					'displayName' => $user->getDisplayName(),
-				]
+				],
 			]);
 			return new TemplateResponse(
-				'social', 'main', 'guest'
+				'social', 'main', []
```

Comment:
Not sure when this changed but we had a separate entrypoint for this back then https://github.com/nextcloud/social/pull/314/files#diff-b5ae6be4c3786d95c10540fefacaac59
https://github.com/nextcloud/social/pull/314/files#diff-b5ae6be4c3786d95c10540fefacaac59

Reply:
hmm I guess you mean what I've removed in #994 right?